docs/metrics.asciidoc

```diff
@@ -60,3 +60,77 @@ This value is normalized by the number of CPU cores and it ranges from 0 to 100%
 
 The Resident Set Size,
 the amount of memory the process occupies in main memory (RAM).
+
+[float]
+[[metric-node.active_handles]]
+=== `node.active_handles`
+
+* *Type:* Long
+* *Format:* Counter
+
+The number of active libuv handles,
+likely held open by currently running I/O operations.
+
+[float]
+[[metric-node.active_requests]]
+=== `node.active_requests`
+
+* *Type:* Long
+* *Format:* Counter
+
+The number of active libuv requests,
+likely waiting for a response to an I/O operation.
+
+[float]
+[[metric-node.cpu.user]]
+=== `node.cpu.user`
```

Comment:
We should discuss with the other agents if this should be put under a shared namespace